src/gc/impl/conservative/gc.d

```diff
@@ -2078,6 +2078,135 @@ struct Gcx
     ToScanStack!(ScanRange) toscanPrecise;
 
     /**
+     * Finds the offset between an index and the next set bit, up to our max.
+     *
+     * Returns: The offset between index and the next bit,
+     *          or -1 if no bits are set between index and max
+     */
+    int bitScanForwardOffset(size_t* bitArray, size_t index, size_t max) nothrow
```

Comment:
Why not just reuse [BitRange](https://github.com/dlang/druntime/pull/1602/files#diff-6d81a130202753008e7b435a4ca71ebcR364) as proposed above?
It's merged by now.